spel/minimal-linux.pkr.hcl

```
@@ -734,6 +741,9 @@ locals {
 
   # Template the description string
   description = "STIG-partitioned [*NOT HARDENED*], LVM-enabled, \"minimal\" %s, with updates through ${formatdate("YYYY-MM-DD", timestamp())}. Default username `maintuser`. See ${var.spel_description_url}."
+
+  # Calculate AWS AMI deprecate_at timestamp
+  aws_ami_deprecate_at = var.spel_duration != null ? timestamp() + var.spel_duration : null
```

Mismatched variable name.
```suggestion
  aws_ami_deprecate_at = var.spel_duration != null ? timestamp() + var.spel_deprecation_lifetime : null
```

---

oops yes, done